poly/main.go

```diff
@@ -114,7 +114,7 @@ func application() *cli.App {
 
 					&cli.StringFlag{
 						Name:  "i",
-						Value: "json",
+						Value: "",
```

Comment:
Removed this default value to avoid triggering an error each time `hash` was called without a pipe.